google/ggrc-core
# GGRC-5128 Batch Comment creation and its mapping to Assessment into single request

# Dependencies

This PR is `on hold` until the dependencies are merged:

- [ ] [GGRC-5042](https://github.com/google/ggrc-core/pull/7814)

# Issue description

Batch Comment creation and its mapping to Assessment into single request

# Steps to test the changes

1) Have assessment with dropdown LCA with mandatory comment.
2) Choose option of dropdown with mandatory comment.
3) Add comment from LCA modal.
4) See network

Expected requests: 
1) PUT for LCA option
2) POST for comment with `related_to`
3) GET for assessment

# Solution description

Remove assessment PUT request with action for mapping comment.

# Sanity checklist




# PR Review checklist
